reports/pharmacy_first_report.Rmd

```diff
@@ -22,6 +22,7 @@ library(gt)
 # Load plotting function
 source(here::here("lib", "functions", "function_tidy_measures.R"))
 source(here::here("lib", "functions", "function_plot_measures.R"))
+source(here::here("lib", "functions", "function_get_validation_data.R"))
```

Comment:
Do we need to source this script here? I think we just need to read the csv output from the script?
```suggestion
```